src/lib/rest/RestService.cpp

```diff
@@ -352,6 +352,89 @@ static void filterRelease(ParseData* parseDataP, RequestType request)
 
 /* ****************************************************************************
 *
+* compCheck - 
+*/
+static bool compCheck(int components, const std::vector<std::string>& compV)
+{
+  for (int ix = 0; ix < components; ++ix)
+  {
+    if (compV[ix] == "")
+    {
+      return false;
+    }
+  }
+
+  return true;
+}
+
+
+/* ****************************************************************************
+*
+* compErrorDetect - 
+*/
+static bool compErrorDetect
+(
+  int                              components,
+  const std::vector<std::string>&  compV,
+  OrionError*                      oeP
+)
+{
+  std::string  details; 
+
+  if ((compV[0] == "v2") && (compV[1] == "entities"))
+  {
+    if ((components == 4) && (compV[3] == "attrs"))  // URL: /v2/entities/<entity-id>/attrs
+    {
+      std::string entityId = compV[2];
+
+      if (entityId == "")
+      {
+        details = "entity id length: 0, min length supported: 1";
+      }
+    }
+    else if ((components == 5) && (compV[3] == "attrs"))  // URL: /v2/entities/<entity-id>/attrs/<attr-name>
+    {
+      std::string entityId = compV[2];
+      std::string attrName = compV[4];
+
+      if (entityId == "")
+      {
+        details = "entity id length: 0, min length supported: 1";
```

Comment:
Now that you have the constant, let's use it in all places, i.e.:

```
details = EMPTY_ENTITY_ID;
```

(Similarly in other `details` set statements in this function)